packages/devtools_app/lib/src/memory/memory_filter.dart

```diff
@@ -139,24 +139,6 @@ class SnapshotFilterState extends State<SnapshotFilterDialog>
     controller = widget.controller;
 
     cancel();
-
-    // Detect and handle checkboxes state changing.
-    addAutoDisposeListener(controller.filterPrivateClassesListenable, () {
-      setState(() {
-        privateClasses.notifier.value = controller.filterPrivateClasses.value;
```

Comment:
are there no other parts of the UI that depend on the value of these notifiers?

Reply:
cleaned up the UI a little further so it is clear that this isn't needed.